tests/coalaJSONTest.py

```diff
@@ -71,7 +71,7 @@ def test_show_all_bears(self):
                 coala.main, 'coala', '--json', '-B', '-I')
             self.assertEqual(retval, 0)
             output = json.loads(stdout)
-            self.assertEqual(len(output['bears']), 8)
+            self.assertEqual(len(output['bears']), 13)
```

Comment:
Eh, why did this grow?  Is this because of your changes in `LocalBearTestHelper.py`, or caused by your other commit which changes `Bear.py`?

I can only see 8 in `tests/test_bears/` and the extra ones are in `TestBearDep`.  but why are they only appearing now, and not before?  Were they hidden intentionally?

I think it would be good to assert the exact list, as that helps show there are multiple levels of dependencies, and different types (some are hidden in the same file).